web/warnings/warnings.go

```diff
@@ -0,0 +1,27 @@
+package warnings
```

Comment:
I would prefer to avoid creating a package in `web` with no HTTP endpoint (and a single function). Maybe we can put it in `web/settings` (or `model/warnings` if you really prefer to have a new package).

Reply:
I'll try `web/settings` since this is web related code. I don't specifically want to create a package but I had issues with circular dependencies and that was the way I found to fix the situation.